tests/classes/array_access_002.phpt

```diff
@@ -134,11 +134,11 @@ ObjectOne::offsetGet(4th)
 int(4)
 ObjectOne::offsetGet(5th)
 
-Notice: Undefined index: 5th in %sarray_access_002.php on line %d
+Notice: Undefined array index "5th" in %s on line %d
```

Comment:
is the distinction between index and offset important? If not, we should probably settle on a single name.

Reply:
I think it's good to keep the distinction between `"5th"` and `5` (with/without quotes), but keeping the index/offset terminology is not. I would actually assume that both index and offset refer to an integer :) Maybe `Undefined array key "5th"`, `Undefined array key 5`.

Reply:
Yess, `array key` is the best choice! :) I also agree with the rest, whether there are quotes or not is a very helpful!